### DISCUSSION TITLE
我想把message类里面的图片保存到本地，请问我应该怎么做

### DISCUSSION THREAD

**Discussion by songelar** [nonebot/discussions#484] [问答]
我似乎没有在文档里面找到有关存储qq图片消息的帮助，cq码的图片链接直接get也不起作用，或者直接将image转成bytesIO也行，只要能将图片数据保存到本地

**Comment by songelar** [comment]
我傻了，我的问题

**Reply by kndxhz** [reply]
老哥请问是怎么解决的

**Reply by songelar** [reply]
> 老哥请问是怎么解决的

后来发现可以直接get，但用httpx库，我用request库有问题，直接get message的url就行，但要把&转义（或者不用？你把链接复制到浏览器能打开就行），具体看shamrock文档，我用的shamrock。
